### PR TITLE
- create a kafka utili library

### PR DESCRIPTION
**Pull request by alberto1ferrari**
- create Asterix actor and connect to kafka
- basic kafka dispatching test

### CODE REVIEW

File: kafka-util/src/main/java/io/airxos/kafka/KafkaActorBootstrapper.java [hunk 1]
```diff
@@ -1,11 +1,9 @@
-package io.aixros.normalization.kafka;
+package io.airxos.kafka;
 
-import com.ge.utm.registry.constant.ModelConstants;
-import com.ge.utm.registry.utils.TopicName;
+import com.ge.utm.registry.utils.Serializer;
```
Comment (denniscammer) [inline kafka-util/src/main/java/io/airxos/kafka/KafkaActorBootstrapper.java]: I think we migrated from using this serializer to the one in core utls.

File: kafka-util/src/main/java/io/airxos/kafka/KafkaPublisher.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package io.airxos.kafka;
+
+import com.ge.utm.registry.utils.Serializer;
```
Comment (denniscammer) [inline kafka-util/src/main/java/io/airxos/kafka/KafkaPublisher.java]: Same comment on the serializer.

File: message_normalization/pom.xml [hunk 3]
```diff
@@ -47,10 +47,12 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
+        <!--
```
Comment (denniscammer) [inline message_normalization/pom.xml]: Should be able to remove. This doesn't have audit logging for the moment.

File: message_normalization/src/main/java/io/aixros/normalization/asterix/service/detail/AsterixService.java [hunk 4]
```diff
@@ -1,20 +1,38 @@
...
+    }
 
-    public AsterixService(){ }
+    @Override
+    public void normalize(String event){
+        //do something here, this is dummy function that just publish the same event on another topic
+        try {
+            fPublisher.publish(event);
+        } catch (ExecutionException e) {
+            e.printStackTrace();
```
Comment (denniscammer) [inline message_normalization/src/main/java/io/aixros/normalization/asterix/service/detail/AsterixService.java]: Move to logging.

Comment (alberto1ferrari) [reply message_normalization/src/main/java/io/aixros/normalization/asterix/service/detail/AsterixService.java]: this is actually just test code, I need to sync up with Anna to see if we anything "real" we want to execute

Comment (denniscammer) [reply message_normalization/src/main/java/io/aixros/normalization/asterix/service/detail/AsterixService.java]: I see. I'm assuming she will take this and put her normalization code on top of it. Right?

**Comment by alberto1ferrari** [comment]
yes correct we should be able to hook the normalization code there instead of just publish whatever we receive.

**Pull request merged**